app/views/alert_mailer/alert_email.html.haml

```diff
@@ -192,6 +192,22 @@
         }
 
   %body(bgcolor="#f6f6f6")
+    - unless @user.alerted_at
```

Comment:
Extract that as simple method? Something like `User#never_alerted?` then you would end up with something that reads better ... `if @user.never_alerted?`

```
def never_alerted?
  alerted_at.nil?
end
```